### PR TITLE
Add support for config_db subscribe and unsubscribe python apis

### PR DESCRIPTION
**Pull request by arlakshm**
Signed-off-by: Arvindsrinivasan Lakshmi Narasimhan <arlakshm@microsoft.com>

The PR https://github.com/Azure/sonic-buildimage/pull/6920 adds the support for swsscommon in `hostcfgd`. The PR test failure are seen because the `config_db.subscribe` doesnt work with swsscommon. This PR adds the support for the following python APIs
 - `config_db.subscribe`
 - `config_db.unsubscribe`

### CODE REVIEW

File: common/configdb.h [hunk 1]
```diff
@@ -126,6 +134,14 @@ class ConfigDBConnector_Native : public SonicV2Connector_Native
             if table in self.handlers:
                 handler = self.handlers[table]
                 handler(table, key, data)
+        
+        def subscribe(self, table, handler):
```
Comment (qiluo-msft) [inline common/configdb.h:138]: >subscribe [](http://example.com/codeflow?start=12&length=9)

Could you add unit test?

Comment (arlakshm) [reply common/configdb.h:156]: added in latest commit

File: tests/test_redis_ut.py [hunk 2]
```diff
@@ -437,3 +437,24 @@ def test_multidb_ConfigDBConnector():
     SonicDBConfig.load_sonic_global_db_config(global_db_config)
     config_db = ConfigDBConnector(use_unix_socket_path=True, namespace='asic1')
     assert config_db.namespace == 'asic1'
+
+def test_ConfigDBSubscribe():
+        table_name = 'Test_table'
+        test_key = 'key1'
+        test_data = {'field1': 'value1'}
+
+        def test_handler(key, data):
```
Comment (qiluo-msft) [inline tests/test_redis_ut.py:446]: >test_handler [](http://example.com/codeflow?start=12&length=12)

Could you also test this handler is really called at least once?

Comment (arlakshm) [reply tests/test_redis_ut.py]: Fixed the test case in the latest commit

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**